src/main/java/org/folio/rest/impl/AbstractHelper.java

```diff
@@ -57,6 +65,45 @@
     setDefaultHeaders();
   }
 
+  /**
+   * Retrieve configuration by moduleName and configName from mod-configuration.
+   *
+   * @param moduleName name of the module for which the configuration is to be retrieved
+   * @param configName name of the configuration to retrieve it
+   * @return CompletableFuture with JsonObject
```

Comment:
This should be `CompletableFuture with List of Config objects` ?

Reply:
Fixed